product_variant_supplierinfo/models/product_supplierinfo.py

```diff
@@ -2,7 +2,7 @@
 # © 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields, api
+from odoo import api, fields, models
 
 
 class ProductSupplierInfo(models.Model):
```

Comment:
The field product_id should have been removed from this module, because it is native starting from v9 https://github.com/odoo/odoo/blob/9.0/addons/product/product.py#L1272 and still native on v10 https://github.com/odoo/odoo/blob/10.0/addons/product/models/product.py#L631 and the definition here is exactly the same as in the native product module.